3-Authorization-II/1-call-api/SPA/src/utils/claimUtils.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Populate claims table with appropriate description
+ * @param {Object} claims ID token claims
+ * @returns claimsObject
+ */
+export const createClaimsTable = (claims) => {
+    let claimsObj = {};
+    let index = 0;
+
+    Object.keys(claims).map((key) => {
+        switch (key) {
```

Comment:
Do we want to add a display for `scp` or would that be overkill?

Reply:
nah this is only for the ID token (which doesn't have the `scp` claim)